ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,1413 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    ! pip install --user jsonlines
</pre><p> </p><p>You should do all your pip installs at the beginning of the notebook, before the kernel restart. </p><p> </p><p>Also, please use the {USER_FLAG} variable to support Google Cloud notebooks.</p><p> </p><p>See notebook_template.ipynb for details</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>Done.</p>